en.md

```diff
@@ -75,13 +75,13 @@ This data is anonymised after a short space of time, generally 7 days. The anony
 
 ### b. Third party integrations
 
-We use a 3rd party service called "HockeyApp" to collect and analyse crash reports of our app. 
+We use a 3rd party service called "App Center" to collect and analyze crash reports of our app.
 
-In particular, HockeyApp saves and processes the following data:
+In particular, App Center saves and processes the following data:
 
  - Technical information which is important for the analysis such as the name of the device, its manufacturer, model, operating system number, app number or SDK version and the device number (IMEI).
 
-HockeyApp is offered under the Microsoft Azure Service Agreement and is operated by: Microsoft Corporation, One Microsoft Way, Redmond, WA 98052-6399, USA. Microsoft is certified under the “Privacy Shield” US-EU data protection agreement and undertakes to uphold the EU data protection provisions. For information on data protection at HockeyApp, go to: [https://hockeyapp.net/imprint/](https://hockeyapp.net/imprint/)
+App Center is offered under the Microsoft Azure Service Agreement and is operated by: Microsoft Corporation, One Microsoft Way, Redmond, WA 98052-6399, USA. Microsoft is certified under the “Privacy Shield” US-EU data protection agreement and undertakes to uphold the EU data protection provisions. For information on data protection at App Center, go to: https://privacy.microsoft.com/privacystatement
```

Comment:
Does it no longer make sense to retain the link to the partner privacy page?

Reply:
@philipecosia

Reply:
Sorry I don't understand the question. For me this change looks fine. Which partner are you referring to? Afaik we are not using hockeyapp anymore at all.

Reply:
Currently, the URL displays as a link (`[https://hockeyapp.net/imprint/](https://hockeyapp.net/imprint/`). This change will make it display as simple text (`https://privacy.microsoft.com/privacystatement`). I ask if that's an intended change for some reasons 👍

Reply:
@philipecosia any idea about the above?